src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -68,31 +68,28 @@ export const setSearchText = createAction(
   'species_selector/set_search_text'
 )<string>();
```

Comment:
I'm confused. Why do you need to change anything about species selector actions in order to update the bookmarks behaviour?

Reply:
I have no idea how this came in. May be linter is the culprit?